File: t.c
#include <stdio.h>

int main()
{
	int i, isprime;
	for(i = 2; i < 100000; i++) {
		int k;
		isprime = 1;
		for(k = 2; k * k <= i; k++) {
			if(i % k == 0) {
				isprime = 0;
				break;
			}
		}
		if(isprime == 1) {
			printf("%d\n", i);
		}
	}
	return 0;
}

File: t.rb
i = 2
while i < 100000
	if i == 2
		puts i.to_s + " is prime"
	elsif i % 2 != 0
		k = 3
		isprime = 1
		while k * k <= i
			if i % k == 0
				isprime = 0
				break
			end
			k += 2
		end
		puts i.to_s + " is prime" if isprime == 1
	end
	i += 1
end
